src/events/ready.ts

```diff
@@ -0,0 +1,15 @@
+import { Client, Events } from "discord.js";
+import Commands from "../Commands";
+
+export default (client: Client): void => {
+    client.on(Events.ClientReady, async() => {
```

Comment:
```suggestion
    client.on(Events.ClientReady, async () => {
```
I think setting up eslint and prettier would be nice. To have consistent formatting.

Reply:
> I think setting up eslint and prettier would be nice. To have consistent formatting.

Definitely, that's a great suggestion. Probably out of scope for this PR so it'd be great to get it going in another one.

Reply:
On second thoughts, I might include it in this PR to make things easier anyway.